src/components/layout/DocsLayout.tsx

```diff
@@ -1,214 +1,136 @@
+import React, { FC } from 'react';
+import { styled } from '@storybook/theming';
+import { global } from '@storybook/design-system';
 import Helmet from 'react-helmet';
-import {
-  SubNav,
-  SubNavTabs,
-  SubNavDivider,
-  SubNavMenus,
-  SubNavRight,
-  SubNavLinkList,
-  styles,
-} from '@storybook/components-marketing';
-import { Container } from '@chromaui/tetra';
+import { Container, color, minMd, minSm } from '@chromaui/tetra';
+import * as ScrollArea from '@radix-ui/react-scroll-area';
 
 import { GLOBAL_SEARCH_IMPORTANCE, GLOBAL_SEARCH_META_KEYS } from '../../constants/global-search';
 import buildPathWithVersion from '../../util/build-path-with-version';
 import GatsbyLinkWrapper from '../basics/GatsbyLinkWrapper';
 import useSiteMetadata from '../lib/useSiteMetadata';
-import { CodeLanguageSelector } from '../screens/DocsScreen/CodeLanguageSelector';
 import { DocsContextProvider } from '../screens/DocsScreen/DocsContext';
-import { RendererSelector } from '../screens/DocsScreen/RendererSelector';
-import { VersionSelector } from '../screens/DocsScreen/VersionSelector';
 import { VersionCTA } from '../screens/DocsScreen/VersionCTA';
+import { Sidebar } from './sidebar/Sidebar';
 
-const { breakpoint, color, pageMargins, spacing } = styles;
 const { GlobalStyle } = global;
 
-const SubNavWrapper = styled.div`
-  background: ${color.lightest};
-  position: sticky;
+const BubblesBackground = styled.img`
+  display: block;
+  position: absolute;
   top: 0;
-  z-index: 2;
+  width: 100%;
+  z-index: -1;
 `;
 
-const Sidebar = styled.div<{ isLoading: boolean }>`
-  display: none;
-  position: relative;
+const Wrapper = styled.div`
+  padding-top: 72px;
 
-  @media (min-width: ${breakpoint * 1.333}px) {
-    display: block;
-    flex: 0 0 240px;
-    margin: 0;
-    padding-bottom: 0;
-    padding-right: 20px;
-    margin-right: 20px;
+  @media (min-width: 440px) {
+    padding-top: 112px;
   }
 
-  ${({ isLoading }) =>
-    isLoading &&
-    css`
-      height: 80vh;
-      border-radius: ${spacing.borderRadius.small}px;
-      ${animation.inlineGlow}
-    `}
+  ${minSm} {
+    display: flex;
+  }
 `;
 
-const ExpandButton = styled(Button)`
-  height: 28px;
-  width: 28px;
-`;
+const SidebarContainer = styled.div`
+  display: none;
 
-const SidebarControls = styled.div`
-  position: absolute;
-  left: -62px;
+  ${minMd} {
+    display: block;
+    position: sticky;
+    top: 112px;
+    align-self: flex-start;
+  }
 `;
 
-const Content = styled.div`
-  flex: 1;
-  min-width: 0; /* do not remove  https://weblog.west-wind.com/posts/2016/feb/15/flexbox-containers-pre-tags-and-managing-overflow */
-  max-width: 800px;
-  margin: 1rem auto 0 auto;
+const SidebarRoot = styled(ScrollArea.Root)`
+  position: relative;
 
-  @media (min-width: ${breakpoint * 1.333}px) {
-    margin-top: 0;
+  ${minMd} {
+    width: 260px;
+    margin: 0;
+    padding-bottom: 0;
+    padding-right: 20px;
+    margin-right: 20px;
+    height: calc(100vh - 112px);
```

Comment:
Does this work with and without the Eyebrow being rendered?

Reply:
No it does not. But how often do you remove the eyebrow?

Reply:
Fair point. We can optimize it later.